auth/src/main/java/com/firebase/ui/auth/ui/credentials/CredentialSaveActivity.java

```diff
@@ -86,10 +71,8 @@ public void onChanged(@Nullable PendingResolution resolution) {
 
     @Override
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-        // Forward activity results to the ViewModel
-        if (!mHandler.onActivityResult(requestCode, resultCode, data)) {
-            super.onActivityResult(requestCode, resultCode, data);
-        }
+        super.onActivityResult(requestCode, resultCode, data);
```

Comment:
Are we sure there's no issue unconditionally calling both?

Reply:
Yeah, as long as we have different request codes (we do: 100 and 101). I felt like we'd get nipped in the behind later if `onActivityResult` was called non-deterministically. (Especially since fragments rely on that call.)